### PR TITLE
Fix sequence padding documentation

### PR DESCRIPTION
**Pull request by joshz123**
The default parameter for padding is pre. The docs reflect the change

### PR Overview

- [ n] This PR requires new unit tests [y/n] (make sure tests are included)
- [ n] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y ] This PR is backwards compatible [y/n]
- [n ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

File: keras_preprocessing/sequence.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def pad_sequences(sequences, maxlen=None, dtype='int32',
     or the length of the longest sequence otherwise.
 
     Sequences that are shorter than `num_timesteps`
-    are padded with `value` at the end.
+    are padded with `value` at the beginning.
```
Comment (Dref360) [inline keras_preprocessing/sequence.py]: or the end if `truncating='post'`

Comment (joshz123) [reply keras_preprocessing/sequence.py]: I will make this change to the PR today. Could you help me understand why the travis CI tests failed so I can fix that as well?

Comment (joshz123) [reply keras_preprocessing/sequence.py]: Did you mean padding?

**Comment by Dref360** [comment]
You need to remove `dist: trusty` in travis.yml

**Comment by joshz123** [comment]
> You need to remove `dist: trusty` in travis.yml

I can do that. What does that line do?

**Comment by Dref360** [comment]
It forces the ubuntu version to be trusty, but we want the latest.
I mad ea PR myself, but I cannot merge it myself, so if you could do it that would be awesome!

**Comment by joshz123** [comment]
Sure. I will do that today. I am trying to be more involved in open source, other than the good first issues page here is there anywhere else that specifically needs work?

**Comment by Dref360** [comment]
At the last Keras board meeting, it was voted to deprecate Keras-preprocessing 
 in favor of a new keras module that would be more composable. (see https://github.com/keras-team/governance/pull/10)

So contributing to this new API would be best. (here: https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python/keras/layers/preprocessing)


The issue right now is that Keras is still tightly integrated with TF, but it should be split in the future! (see https://github.com/tensorflow/community/pull/202)

This will make contributing easier because the CI will be faster.

**Comment by joshz123** [comment]
Thanks for your response! A standalone Keras repo would be great because I definitely feel more comfortable contributing to Keras than most of TensorFlow and it is easy to get lost or have difficulty navigating such a large repository. I will migrate future contributions to the TensorFlow repo
